tests/unit/test_ringcells.py

```diff
@@ -143,3 +145,51 @@ def test_coreneuron(create_tmp_simulation_config_file):
     assert coreneuron_data.is_dir() and not any(coreneuron_data.iterdir()), (
         f"{coreneuron_data} should be empty."
     )
+
+
+@pytest.mark.parametrize(
```

Comment:
can we apply the stimulus to something that is not the soma? In that case may we have a test on another section and test the same? The soma is often special. 

Otherwise, discard this comment

Reply:
We can add an additional test in the near future. I also want more tests on the number of spikes when using this big soma. I didn't want to drag out getting the fix in for reverting the simulation behavior